src/modules/75-cf/components/FlagActivation/ServicesList.module.scss

```
@@ -14,6 +14,15 @@
   width: -webkit-fill-available;
 }
 
+.searchBar {
+  width: calc(100% - 30%);
```

You don't really need to `calc` this. I'm pretty sure 100% - 30% is 70%
```suggestion
  width: 70%;
```